server/map_rpc_server.go

```diff
@@ -145,6 +145,9 @@ func (t *TrillianMapServer) getLeavesByRevision(ctx context.Context, mapID int64
 	for i, l := range leaves {
 		leavesByIndex[string(l.Index)] = &leaves[i]
 	}
+	if len(indices) != len(leavesByIndex) {
+		glog.Infof("%v: request had %v indices, %v of these are unique", mapID, len(indices), len(leavesByIndex))
```

Comment:
That should be a verbose level log I think.

Reply:
Yup, this could get quite spammy. Updated in subsequent commit.